src/playturn_network_adapter.cpp

```diff
@@ -86,15 +86,15 @@ bool playturn_network_adapter::read(config& dst)
 	}
 	config& child = dst.add_child(next_->key);
 	//TODO: implement a non const version of ordered children
```

Comment:
Remove this comment. There are now non-const versions of `config::ordered_begin()` and `config::ordered_end()`. In fact, the lack of those overloads was the reason why these `const_cast`s were originally needed!